static/js/theme-map/Maps/Map.js

```diff
@@ -53,6 +53,15 @@ class MapOptions {
     this.providerOptions = {};
     this.singlePinZoom = 14;
     this.wrapper = null;
+    this.language = 'en';
+  }
+
+  /**
+   * @param {string} language The language displayed on map
+   */
+  withLanguage(language) {
+    this.language = language;
```

Comment:
I would just like to check is this language always a 2 character language like "es", or can it be a locale such as "es_US"? If it's just 2 chars then maybe we should take a substring? See my pr here: https://github.com/yext/answers-search-ui/pull/1070/files

Reply:
this language would be from `getLanguageForProvider` function (from this [techop fix](https://github.com/yext/answers-hitchhiker-theme/pull/891/files)). It will not always be just 2 chars but it should be appropriate for the corresponding provider. In this case with Mapbox, it will always be 2 chars since we don't have any specific supported language locale specified for Mapbox

Reply:
Okay, in that case what do you think of calling this `withLocale` instead since we generally use "language" to refer to just the two char language, and a "locale" to include additional info such as a dialect or region

Reply:
sure thing. Updated!